utils/route.tsx

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {ErrorPageTypes} from 'utils/constants';
+
+import {isGuest} from 'utils/utils.jsx';
+
+interface ICallback {
+    (error: Error | null, comp?: string | Component[]): unknown;
+}
+
+interface Component {
+    default: string;
+}
+
+export function importComponentSuccess(callback: ICallback): unknown {
```

Comment:
I just looked through the code and turns out we aren't using `importComponentSuccess`  or `createGetChildComponentsFunction` anywhere. Instead of trying to type these, can we just remove them?

Reply:
done